From d3fd022fb0886f724f8e6a9e70e2f29503534093 Mon Sep 17 00:00:00 2001
From: Madhu <enometh@net.meer>
Date: Sun, 10 Dec 2023 14:30:18 +0530
Subject: [PATCH] USE=ffmpeg: fix compilation with ffmpeg-6

* modules/media/gtkffmediafile.c: (gtk_ff_media_file_open_audio_device):
  don't try to use removed fields of AVOutputFormat
---
 modules/media/gtkffmediafile.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/modules/media/gtkffmediafile.c b/modules/media/gtkffmediafile.c
index 2839cb8..e2643d2 100644
--- a/modules/media/gtkffmediafile.c
+++ b/modules/media/gtkffmediafile.c
@@ -847,6 +847,7 @@ gtk_ff_media_file_open_audio_device (GtkFfMediaFile *self)
   const AVOutputFormat *candidate;
   int errnum;
 
+#if 0
   /* Try finding an audio device that supports setting the volume */
   for (candidate = av_output_audio_device_next (NULL);
        candidate != NULL;
@@ -855,6 +856,7 @@ gtk_ff_media_file_open_audio_device (GtkFfMediaFile *self)
       if (candidate->control_message)
         break;
     }
+#endif
 
   /* fallback to the first format available */
   if (candidate == NULL)
-- 
2.39.2.101.g768bb238c4

